app/src/main/java/chat/rocket/android/fragment/sidebar/SidebarMainFragment.java

```diff
@@ -10,19 +10,7 @@
 import android.support.v7.widget.SearchView;
 import android.view.View;
 import android.widget.CompoundButton;
-import android.widget.ImageView;
```

Comment:
Thanks for your review @luciofm ! I always do `cmd + O` to remove unnecessary imports and maybe this action reorders it.

We really need a default config and coding style.